scalafix-core/shared/src/main/scala/scalafix/internal/config/DisableSyntaxConfig.scala

```diff
@@ -0,0 +1,191 @@
+      case KwFinal()     => Some("final")
+      case KwFinally()   => Some("finally")
+      case KwFor()       => Some("for")
+      case KwForsome()   => Some("forSome")
+      case KwIf()        => Some("if")
+      case KwImplicit()  => Some("implicit")
+      case KwImport()    => Some("import")
+      case KwLazy()      => Some("lazy")
+      case KwMatch()     => Some("match")
+      case KwMacro()     => Some("macro")
+      case KwNew()       => Some("new")
+      case KwNull()      => Some("null")
+      case KwObject()    => Some("object")
+      case KwOverride()  => Some("override")
+      case KwPackage()   => Some("package")
+      case KwPrivate()   => Some("private")
+      case KwProtected() => Some("protected")
+      case KwReturn()    => Some("return")
+      case KwSealed()    => Some("sealed")
+      case KwSuper()     => Some("super")
+      case KwThis()      => Some("this")
+      case KwThrow()     => Some("throw")
+      case KwTrait()     => Some("trait")
+      case KwTrue()      => Some("true")
+      case KwTry()       => Some("try")
+      case KwType()      => Some("type")
+      case KwVal()       => Some("val")
+      case KwVar()       => Some("var")
+      case KwWhile()     => Some("while")
+      case KwWith()      => Some("with")
+      case KwYield()     => Some("yield")
+      case _             => None
+    }
+  }
+  private val keywords = List(
+    "abstract",
+    "case",
+    "catch",
+    "class",
+    "def",
+    "do",
+    "else",
+    "enum",
+    "extends",
+    "false",
+    "final",
+    "finally",
+    "for",
+    "forSome",
+    "if",
+    "implicit",
+    "import",
+    "lazy",
+    "match",
+    "macro",
+    "new",
+    "null",
+    "object",
+    "override",
+    "package",
+    "private",
+    "protected",
+    "return",
+    "sealed",
+    "super",
+    "this",
+    "throw",
+    "trait",
+    "true",
+    "try",
+    "type",
+    "val",
+    "var",
+    "while",
+    "with",
+    "yield"
+  )
+  private val keywordsSet = keywords.toSet
+
+  // per file:
+  //   Space    Space
+  //   Tab      Tab
+  //   CR       CarriageReturn
+  //   LF       LineFeed
+  //   FF       FormFeed
+
+  implicit val reader: ConfDecoder[DisabledKeyword] =
+    new ConfDecoder[DisabledKeyword] {
+      override def read(conf: Conf): Configured[DisabledKeyword] = {
+        def readKeyword(keyword: String): Configured[DisabledKeyword] = {
+          if (keywordsSet.contains(keyword))
+            Configured.Ok(DisabledKeyword(keyword))
+          else Configured.NotOk(oneOfTypo(keyword, conf))
+        }
+        conf match {
+          case Conf.Null() => readKeyword("null")
+          case Conf.Str(keyword) => readKeyword(keyword)
+          case Conf.Bool(value) =>
+            if (value) readKeyword("true")
+            else readKeyword("false")
+
+          case _ => Configured.typeMismatch("String", conf)
+        }
+      }
+    }
+
+  // XXX: This is from metaconfig.ConfError
```

Comment:
Why XXX?

Reply:
https://softwareengineering.stackexchange.com/a/65470/38296

it is used as a marker for code that requires attention.